test/service_manager_test.dart

```diff
@@ -0,0 +1,144 @@
+@TestOn('vm')
+import 'dart:async';
+import 'dart:io';
+import 'package:devtools/eval_on_dart_library.dart';
```

Comment:
move all the package:devtools deps to bellow the other dependencies with a line break
as those deps are from this package.

Reply:
Done.